3-extra/card-validator.js

```diff
@@ -0,0 +1,32 @@
+//creating card validator function
+
+function creditCardNumberValidator(cardNumber) {
+  let cardNumberElement = cardNumber.toString().split("");
+  if (!cardNumberElement.every((element) => /[0-9]/.test(element))) {
+    return false + "Your card is invalid!";
```

Comment:
The function should return a boolean, so just `false`, no need to add a string to it.